packages/muelu/utils/misc/setSshEnv.python

```diff
@@ -62,10 +68,10 @@ for agent in agentList:
         #print "checking key=>"+key+"<"
         if keyFingerprint in key:
           keyFound=1
-          print envCmd + agent
+          print (envCmd + myagent)
```

Comment:
You might need to add `from __future__ import print_function` to the top of the script.

Reply:
done -- python is weird